lib/FusionInventory/Agent/Task/Inventory/MacOS/CPU.pm

```diff
@@ -53,7 +53,8 @@ sub _getCpus {
     }
     close $handle;
 
-    my $type  = $sysprofile_info->{'Processor Name'} ||
+    my $type  = $sysctl_info->{'machdep.cpu.brand_string'} || 
```

Comment:
For BSD you can update BSD/CPU.pm.
But I was asking you to use trimWhitespace() method to clean up the retrieved value as for example it compresses white spaces. I don't see why you're talking about BSD then.